backend/prisma/schema.prisma

```diff
@@ -22,11 +22,35 @@ model event {
   description String
   image       String
   category    String
+  comments    comment[]
+  user        user     @relation(fields: [userId], references: [id])
+  userId      Int
+  date        String
+  time        String
```

Comment:
Use Datetime data type

Reply:
**_To do:_** 
- Use Datetime
- Learn how to ask for Datetime in a form